MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Grouping/IMvxGroupableRecyclerAdapter.cs

```diff
@@ -0,0 +1,9 @@
+using System.Windows.Input;
+
+namespace MvvmCross.Droid.Support.V7.RecyclerView.Grouping
+{
+    public interface IMvxGroupableRecyclerAdapter
+    {
+        ICommand GroupHeaderClickCommand { get; set; }
```

Comment:
done in https://github.com/MvvmCross/MvvmCross/pull/1568/commits/96b21f303d65024ce833287a7ffc5fec635028e0